api/events/backend_events/djListCycle.md

```diff
@@ -1,12 +1,17 @@
 # djListCycle message
 
+<<<<<<< HEAD
```

Comment:
Hey, thanks for PR,

Unfortunately I cannot accept a PR with a conflict header in it, such as this.